src/intervals/reduction.jl

```diff
@@ -0,0 +1,45 @@
+# This file is part of the IntervalArithmetic.jl package; MIT licensed
+
+"""
+The file contains the reduction functions stated in the section 12.12.12 with correctly rounded results.
+r is the rounding direction which takes 0.0 for RoundingToZero , 0.5 for RoundingNearest , +Inf for RoundingUp and -Inf for RoundingDown.
+"""
+function mpfr_vector_sum(v :: Vector{T} , r :: Float64) where T
```

Comment:
I would remove `mpfr` from the name of these functions -- that is an implementation detail.